airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/factory/DataSourceFactory.java

```diff
@@ -272,6 +278,8 @@ public DataSource build() {
        */
       config.setInitializationFailTimeout(Integer.MIN_VALUE);
 
+      config.setConnectionInitSql(connectionInitSql);
```

Comment:
I should probably remove the `RedshiftDestinationHandler#execute` and test with this to see SET var works consistently